javascript/soyutils_velog.js

```diff
@@ -92,10 +106,10 @@ class Metadata {
 // NOTE: we need to use toLowerCase in case the xid contains upper case
 // characters, browsers normalize keys to their ascii lowercase versions when
 // accessing attributes via the programmatic APIs (as we do below).
-/** @package */ const ELEMENT_ATTR = 'data-' + xid('soylog').toLowerCase();
+/** @package */ const ELEMENT_ATTR = 'data-' + xidGen('soylog').toLowerCase();
 
 /** @package */ const FUNCTION_ATTR =
-    'data-' + xid('soyloggingfunction').toLowerCase() + '-';
+    'data-' + xidGen('soyloggingfunction').toLowerCase() + '-';
```

Comment:
it's used in two places in `soy_velog`. to expose these IDs, the `xidGen` function is exported as `exports.xid`, which would allow the following syntax to access the generator:

```js
const {xid} = goog.require('soy.velog');
// ...
const soylog = someElement.getAttribute('data-' + xid('soylog').toLowerCase());
```